pkgs/development/python-modules/shub/default.nix

```diff
@@ -0,0 +1,42 @@
+{ buildPythonPackage, fetchPypi
+, click, docker-py, pip, pyyaml, retrying, requests
+, scrapinghub, tqdm, six, tox, pytest
+, glibcLocales, lib
+}:
+
+buildPythonPackage rec {
+    pname = "shub";
+    version = "2.8.2";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "10v8hh21jjc16hhsjzcxqmjpysz5ans7pm5pcxizj6029mjkm3ix";
+    };
+
+    propagatedBuildInputs = [ 
+      click
+      docker-py
+      pip
+      pyyaml
+      retrying
+      requests
+      scrapinghub
+      tqdm
+      six
+    ];
+    
+    # Tests are network-based
+    doCheck = false;
+    
+    # Prevent unicode decoding error in setup.py
```

Comment:
same

Reply:
To be honest, I don't see it anymore either :/